### ISSUE TITLE
[Bug Report] [Vue warn]: Invalid prop: type check failed for prop "cell". Expected Array, got Object

### ISSUE THREAD

**Issue by chengweiou** [element-plus/element-plus#4184]
### Element Plus version
1.2.0-beta.1

### OS/Browsers version
Version 95.0.4638.54 (Official Build) (x86_64)

### Vue version
3.2.21

### Reproduction Link
https://jsfiddle.net/46yncqhg/

### Steps to reproduce
just add a <el-date-picker/>. the warning will show in console.

### What is Expected?
no warning

### What is actually happening?
```
[Vue warn]: Invalid prop: type check failed for prop "cell". Expected Array, got Object  
  at <ElDatePickerCell cell= {row: 5, column: 6, type: 'next-month', inRange: undefined, start: undefined, …} > 
  at <BasicDateTable key=0 selection-mode="day" date= M3 {$L: 'en', $u: undefined, $d: Tue Nov 02 2021 23:54:41 GMT-0700 (Pacific Daylight Time), $x: {…}, $y: 2021, …}  ... > 
  at <PanelDatePick visible=false actualVisible=false parsedValue=undefined  ... > 
  at <BaseTransition onAfterEnter=fn<onAfterEnter> onAfterLeave=fn<onAfterLeave> onBeforeEnter=fn<onBeforeEnter>  ... > 
  at <Transition name="el-zoom-in-top" onAfterEnter=fn<onAfterEnter> onAfterLeave=fn<onAfterLeave>  ... > 
  at <ElPopper ref="refPopper" visible=false onUpdate:visible=fn  ... > 
  at <Picker valueFormat="YYYY-MM-DD" name="" popperClass=""  ... > 
  at <ElDatePicker modelValue="" onUpdate:modelValue=fn type="date"  ... > 
  at <ElFormItem label="日期" prop="dateAt" style= {margin-right: '100px', flex: '1 0 500px'} > 
  at <ElForm ref="formEl" style= {flex: '1'} model= {id: null, person: {…}, age: 0, height: 0, weight: 0, …}  ... >
```

**Comment by msidolphin** [comment]
already fixed #4134

**Comment by chengweiou** [comment]
@msidolphin I tried delete my nodemodules and installed again. it still showing. Please check
